package.json

```diff
@@ -4,6 +4,7 @@
   "dependencies": {
```

Comment:
- [ ] It would be better if our app can start without throwing any errors. Let us fix it and submit another code review request :man_technologist: 
![Screenshot from 2022-09-28 00-36-01](https://user-images.githubusercontent.com/46388113/192661648-149cdc05-ebe7-4d8d-a746-23c2df56644e.png)